account_cancel_with_reversal/tests/test_account_move_reverse.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+from datetime import date
+from openerp.tests import common
+
+
+class TestAccountInvoiceCancel(common.TransactionCase):
```

Comment:
Change name to TestAccountCancelWithReversal(common.TransactionCase)
It is not only invoice, but also voucher.